### PR TITLE
Add 'Explore & Build' section

### LINKED ISSUE #32: Explore section
- [x] Background & Spacing
- [x] Explore description block
- [x] Expanding numbered component (same component for each block)
- [x] Icons should be local SVG icons (final svg to come, use placeholders if needed)
- [ ] Accessibility review for Explore
- [x] Accordion: Click on a section and it expands while closing all other sections.
    - [x] Accordion Component
    - [x] Click on the section to expand
    - [x] Click on the “->” and go to the page on Docs.frequency.xyz (target _blank)

Section URLs
- [ ] Become a provider: [https://docs.frequency.xyz/Guides/BecomeAProvider.html](https://docs.frequency.xyz/Guides/BecomeAProvider.html)
- [ ] Gateway: [https://docs.frequency.xyz/Guides/Gateway.html](https://docs.frequency.xyz/Guides/Gateway.html)
- [ ] Add Single Sign-on to your App: [https://docs.frequency.xyz/Guides/SSO.html](https://docs.frequency.xyz/Guides/SSO.html)
- [ ] Frequency Core: [https://docs.frequency.xyz](https://docs.frequency.xyz) 

Tasks Dependent on this:
-  #33 

Helpful example:
- Figma: https://www.figma.com/design/EjFt1XdacOogDZoQ7RetrI/Frequency-Style-Guidelines?node-id=3416-70582&node-type=frame&t=Y7Sku83pGxNxKVbj-0
- Example toggle: https://www.figma.com/proto/EjFt1XdacOogDZoQ7RetrI/Frequency-Style-Guidelines?page-id=3267%3A62157&node-id=4060-871&node-type=frame&viewport=-21220%2C-7332%2C0.25&t=kijLfxY9Fvk612m2-1&scaling=scale-down&content-scaling=fixed&starting-point-node-id=4060%3A687
- https://www.radix-ui.com/primitives/docs/components/accordion
- https://frequency-chain.github.io/style-guide/?path=/story/ui-components-features-accordionlist--accordion-list-19-ba-35-e-6
Image

### PR DESCRIPTION
**Pull request by JoeCap08055**
# Description

Initial implementation of "Explore & Build" section

- Closes #32 

## Screenshot(s)
image

image

image

image

image

- [ ] Content Change (Required approval in Slack: `#frequency-xyz` and remember that changes along the way trigger
      re-approval.)
  - [ ] Approved HE (Required for technical wording)
  - [ ] Approved CM
  - [ ] Approved UO
- [ ] New Feature

# How to Test?

Please describe how to test that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

1. Pull and run locally
2. Go to "Dev Docs" link in header
3. See that the "Explore & Build" section matches the design & behaves as per #32 

# Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code & PR, particularly in hard-to-understand areas
- [ ] I have checked at all the breakpoints to make sure it works on all screen sizes

### CODE REVIEW

**Review by wilwade — DISMISSED** [review]
Needs to be merging into `developer-portal` branch not `main`

File: src/components/HAccordion.svelte [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+  import { createEventDispatcher, type Snippet } from 'svelte';
+  import { Button } from '@frequency-chain/style-guide';
+  import Vector from '../lib/assets/Vector.svg';
+
+  const dispatch = createEventDispatcher();
+
+  interface HAccordionProps {
+    state: 'open' | 'closed';
```
Comment (claireolmstead) [inline src/components/HAccordion.svelte:9]: Is there anything in the way of us using `isOpen` and using a boolean so we don't have to do so much equals logic?

File: src/components/Header/HeaderNavMobile.svelte [hunk 2]
```diff
@@ -10,11 +10,11 @@
   class={`z-1 fixed right-0 top-0 z-10 w-[100vw] flex-col justify-between gap-6 overflow-y-scroll bg-navy px-[12%] font-title transition-[height] duration-[1s] sm:flex md:hidden lg:hidden ${isOpen ? 'h-[100vh]' : 'h-0'}`}
   aria-expanded={isOpen}
 >
-  <nav id="mobile-navigation" aria-label="Main" class="mt-[60px] flex flex-col gap-4 text-2xl uppercase text-cream">
+  <nav id="mobile-navigation" aria-label="Main" class="text-cream mt-[60px] flex flex-col gap-4 text-2xl uppercase">
     <NavItems textColor="cream" bind:section isDesktop={false} />
   </nav>
 
-  <div class="z-50 pb-[40px] font-sans text-cream">
+  <div class="text-cream z-50 pb-[40px] font-sans">
```
Comment (claireolmstead) [inline src/components/Header/HeaderNavMobile.svelte:17]: double checking that cream is still a value being used. I know it was taken out of the tailwind config, so currently I don't think this is doing anything.

File: src/components/OpenCloseIcon.svelte [hunk 3]
```diff
@@ -16,14 +16,14 @@
       y1="50%"
       x2="100%"
       y2="50%"
-      class={`origin-left translate-y-[20%] stroke-[10%] transition-all duration-[0.3s] ${isOpen ? 'translate-x-[8px] translate-y-[35%] -rotate-45 stroke-cream' : 'stroke-indigo rotate-0'}`}
+      class={`origin-left translate-y-[20%] stroke-[10%] transition-all duration-[0.3s] ${isOpen ? 'stroke-cream translate-x-[8px] translate-y-[35%] -rotate-45' : 'stroke-indigo rotate-0'}`}
```
Comment (claireolmstead) [inline src/components/OpenCloseIcon.svelte:19]: same note here. change to `white`

File: src/lib/assets/icon-provider.svg
Comment (claireolmstead) [inline src/lib/assets/icon-provider.svg:1]: are we sure this box should be blue? seems weird since the background is blue. maybe flag with Heather?

Comment (wilwade) [reply src/lib/assets/icon-provider.svg:1]: These aren't the final icons, just the placeholders

File: src/routes/docs/+page.svelte [hunk 4]
```diff
@@ -1,5 +1,7 @@
 <script lang="ts">
   import DocsMain from './components/DocsMain.svelte';
+  import Explore from './components/Explore.svelte';
 </script>
 
 <DocsMain />
+<Explore classes="mt-4" />
```
Comment (claireolmstead) [inline src/routes/docs/+page.svelte:7]:
```suggestion
<Explore />
```
This is currently not doing anything and I don't think it's necessary. if we want to add spacing, i would do it inside of the component or at the top level using something like `gap`

File: src/routes/docs/components/Explore.svelte [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+<div
+  id="explore_section"
+  class="relative mt-f96 flex w-[100vw] min-w-full flex-row bg-navy sm:px-f64 sm:py-f80 lg:px-f128 lg:py-f96 vertical-lg:min-h-[550-px] vertical-lg:w-[100%]"
+>
+  <div id="explore_description" class="relative flex flex-row bg-navy pr-f12 text-teal vertical-lg:min-h-[550px]">
+    <div class="flex flex-col gap-f32">
+      <span class="h1 mt-f8 text-clip text-wrap font-title font-normal">Explore &amp; Build</span>
+
+      <div class="sm text-white sm:max-w-[515px] sm:pb-[100px] lg:pb-[200px]">
```
Comment (claireolmstead) [inline src/routes/docs/components/Explore.svelte:29]:
```suggestion
      <div class="text-sm text-white sm:max-w-[515px] sm:pb-[100px] lg:pb-[200px]">
```
I take back my comment from earlier. The line height in the designs is larger that what's defined in the SG. By just setting the text size, the line height is left alone.

File: src/routes/docs/components/Explore.svelte [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+            <li>Add decentralized authentication and onboarding workflows</li>
+            <li>Connect your users with their universal social graph</li>
+            <li>Read, write, and interact with social media content</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="flex flex-row lg:min-w-[716px]">
```
Comment (claireolmstead) [inline src/routes/docs/components/Explore.svelte:46]: Add comment explaining out min width?

File: src/style/app.css [hunk 7]
```diff
@@ -35,7 +35,7 @@
 
   /* Layout */
   .freq-container {
-    @apply mx-auto w-full sm:px-[30px] md:px-[40px] lg:max-w-[100vw] lg:px-[80px] xl:max-w-page;
```
Comment (claireolmstead) [inline src/style/app.css:38]: I think we still want `mx-auto` to center content.

**Review by claireolmstead — APPROVED** [review]
Paired, read through code and tested code. Looking good. Added one commit to fix the spacing issues with the width of the explore div in extra wide screens. Feel free to test further and edit if you see fit.

**Comment by claireolmstead** [comment]
Looks like there are some conflicting peer deps with the upgrade to svelte 5.

File: src/style/app.css (hunk 7 above)
Comment (JoeCap08055) [reply src/style/app.css:38]: I think this was a rebase conflict error; I'll put it back.

File: package.json [hunk 8]
```diff
@@ -10,6 +10,7 @@
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
     "lint": "prettier --check . && eslint .",
+    "lint:fix": "prettier --write . && eslint --fix .",
```
Comment (shannonwells) [inline package.json:13]: is there a reason to have both lint:fix and format?

Comment (wilwade) [reply package.json:13]: I've often added both just to make it easy to find. I tend toward `format` (Rust and Go style perhaps?), but others have tended toward `lint:fix`. I'm find with standardizing on one or the other.

Comment (JoeCap08055) [reply package.json:13]: eh.
`format` was already there, but `lint:fix` was missing. Maybe `lint:fix` should *only* fix lint-y problems?

**Pull request merged**